package.json

```diff
@@ -21,8 +21,12 @@
   },
   "scripts": {
     "prepare": "husky install",
-    "format": "prettier **/*.js --write",
-    "lint": "eslint . --cache --fix",
+    "format": "prettier --write .",
+    "test:eslint": "eslint . --cache",
+    "test:prettier": "prettier --check **/*.js",
+    "test:typescript": "npm run type",
```

Comment:
maybe convert this to run `tsc` directly and remove the `type` script?

Reply:
We still need `type` for the build step, so I'm going to leave this as is.